eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/ProtocolConstant.java

```diff
@@ -19,7 +19,6 @@
 
 public final class ProtocolConstant {
 
-    public static final String CE_PROTOCOL = "cloudevents";
     public static final String EM_MESSAGE_PROTOCOL = "eventmeshmessage";
     public static final String OP_MESSAGE_PROTOCOL = "openmessage";
     public static final String PROTOCOL_DESC = "http";
```

Comment:
Another `cloudevents` removed but `eventmeshmessage` and `openmessage` remaining here. The protocols are torn apart and still duplicate.